src/utils/datetime/formatters.ts

```diff
@@ -804,7 +804,7 @@ export const convertDateToRFC3339 = (date: Date, timeZone: string): string => {
     const localTime = timeStringParsed[0]
     const utcOffset = timeStringParsed[1].replace('GMT', '')
 
-    return `${year}-${month}-${dayOfMonth} ${localTime}${utcOffset}`
+    return `${year}-${month}-${dayOfMonth}T${localTime}${utcOffset}`
```

Comment:
is there a test case we can add to this?

Reply:
Yes, just pushed it up as additional change.